x-pack/plugins/file_upload/public/components/geojson_upload_form/index_name_form.tsx

```diff
@@ -34,15 +34,19 @@ export class IndexNameForm extends Component<Props> {
     const indexName = event.target.value;
     this.props.onIndexNameChange(indexName);
     this._validateIndexName(indexName);
-    this.props.onIndexNameValidationStart();
+    if (this.props.onIndexNameValidationStart) {
+      this.props.onIndexNameValidationStart();
+    }
   };
 
   _validateIndexName = _.debounce(async (indexName: string) => {
     const indexNameError = await validateIndexName(indexName);
     if (!this._isMounted || indexName !== this.props.indexName) {
       return;
     }
-    this.props.onIndexNameValidationEnd();
+    if (this.props.onIndexNameValidationEnd) {
```

Comment:
If check for onIndexNameValidationEnd is not needed since onIndexNameValidationEnd is required.